### ISSUE TITLE
Fix execption in WrFilesystemListFragment

### ISSUE THREAD

**Issue by gsantner** [gsantner/markor#376]
report from gplay below

![grafik](https://user-images.githubusercontent.com/6735650/46469249-26eea100-c7d3-11e8-8daa-1c2f99825365.png)

**Comment by MikeMikerson** [comment]
I'd like to do this one, please.

**Comment by gsantner** [comment]
Theres enough for everybody! :D. Thanks for helping the project much, always welcome. You can always go for issues you don't need a permission or like that ;).

**Comment by MikeMikerson** [comment]
Thanks! The issue was caused in the Notebook tab. The exception happens when you click the rename menu item. I reproduced the error by doing the following:

* Long click an item
* When the "Select entries" menu comes up, press the FAB.
* Go back to the tab. Although no items are selected, the "Select entries" menu will be the same.
* Press the rename menu item without anything selected.

I fixed this issue in pull request #386.
